tools/flang1/flang1exe/exterf.c

```diff
@@ -1306,7 +1306,8 @@ export_ivl_ict(int lineno, VAR *ivl, ACL *ict, int dostructures)
 static void
 export_component_init(int cleanup)
 {
-  int dtype, flag;
+  unsigned int dtype;
+  int flag;
```

Comment:
it's not often to see signed flags... is it really ok?

Reply:
This code is a bit weird. `flag` is bitwise AND-ed and OR-ed with `ict->ci_exprt` in the function, but the `ci_exprt` field is declared as a single bit in a bit field. I am not sure whether initializing `flag` to 2 in this function is intentional or not. 

I will undo this hunk, since `dtype` should not be unsigned. The warning is coming from a comparison with `stb.dt.stg_avail` which is declared unsigned by the `STG_MEMBERS` macro.